### DISCUSSION TITLE
Wondering where I could find the JSON schema generated by trivy

### DISCUSSION THREAD

**Discussion by Sudo-Guy** [aquasecurity/trivy#7150] [Q&A] [answered]
### Question

I am currently trying to generate a pydantic model from the misconfiguration JSON reports produced by Trivy, I was wondering where I could find a JSON schema.

### Target

Filesystem

### Scanner

Misconfiguration

### Output Format

JSON

### Mode

Standalone

### Operating System

linux (~various distros)

### Version

```bash
Version: 0.52.2
Vulnerability DB:
  Version: 1
  UpdatedAt: 2023-02-08 12:48:16.989777838 +0000 UTC
  NextUpdate: 2023-02-08 18:48:16.989777438 +0000 UTC
  DownloadedAt: 2024-06-27 14:55:48.412815 +0000 UTC
Check Bundle:
  Digest: sha256:ef2d9ad4fce0f933b20a662004d7e55bf200987c180e7f2cd531af631f408bb3
  DownloadedAt: 2024-06-27 15:30:18.17963 +0000 UTC
```

**Comment by knqyf263** [comment]
Do you mean the JSON schema of the result JSON?

**Reply by Sudo-Guy** [reply]
> Do you mean the JSON schema of the result JSON?

Sorry for the late reply, but yes.

**Reply by knqyf263** [reply]
We don't have it now. But it makes sense adding it.

**Reply by simar7** [reply, marked as answer]
@Sudo-Guy this is the Go struct which is marshaled into the JSON result you see https://github.com/aquasecurity/trivy/blob/8d618e48a2f1b60c2e4c49cdd9deb8eb45c972b0/pkg/types/report.go#L109
